bigframes/core/nodes.py

```diff
@@ -523,6 +524,7 @@ def from_table(table: bq.Table, columns: Sequence[str] = ()) -> GbqTable:
             table_id=table.table_id,
             physical_schema=schema,
             n_rows=table.num_rows,
+            is_view=(table.table_type == "VIEW"),
```

Comment:
I wonder if EXTERNAL and MATERIALIZED_VIEW have the same problem. https://cloud.google.com/python/docs/reference/bigquery/latest/google.cloud.bigquery.table.Table#google_cloud_bigquery_table_Table_table_type

Reply:
As chatted offline, MATERIALIZED_VIEW will be skipped for now. 

And external table seems also shows row count zero if we read num_rows.

Logic updated.